src/mountfsd/mountwork.c

```diff
@@ -484,7 +540,7 @@ static int vl_method_mount_image(
                 r = dissect_loop_device(
                                 loop,
                                 &verity,
-                                /* mount_options= */ NULL,
+                                p.options,
```

Comment:
hu, this is not ok. option strings must come at a higher security requirement, i.e. you must upgrade the polkit action, if you see any.

Reply:
What about allowing the few known-good ones without privs, and require privs for the rest? EG I think it would be nice to allow specifying noexec, nosuid and a couple others that restrict usage

Reply:
The use case is making the service options work for user services too as they currently do for the system services - RootImageOptions, etc etc, for feature parity